corehq/apps/dashboard/models.py

```diff
@@ -342,3 +343,41 @@ def _get_release_manager_url(app):
                                None,  # full_name
                                _get_release_manager_url(a),
                                self.secondary_url_icon) for a in apps]
+
+
+class DataPaginatedContext(BasePaginatedTileContextProcessor, ExportsPermissionsMixin):
+    """Generates the Paginated context for the Data Tile."""
+    domain = None
+
+    def __init__(self, tile_config, request, in_data):
+        self.domain = request.domain
+        super(DataPaginatedContext, self).__init__(tile_config, request, in_data)
+
+    @property
+    def total(self):
+        return len(self.form_exports) + len(self.case_exports)
+
+    @property
+    @memoized
+    def form_exports(self):
+        exports = []
+        if self.has_edit_permissions:
+            exports = FormExportSchema.get_stale_exports(self.request.domain)
+        return exports
+
+    @property
+    @memoized
+    def case_exports(self):
+        exports = []
+        if self.has_edit_permissions:
+            exports = CaseExportSchema.get_stale_exports(self.domain)
+        return exports
+
+    @property
+    def paginated_items(self):
+        for export in self.form_exports + self.case_exports:
+            yield self._fmt_item(
+                export.name,
+                reverse('export_download_{}s'.format(export._default_type),
```

Comment:
I don't think we should rely on this url pattern working this way (and I don't think we should access the private `_default_type` variable from outside the class. Also this would fail a grep test for that url pattern which I think is not ideal.

Maybe you can just change to `'export_download_cases' if isinstance(export, CaseExportSchema) else 'export_download_forms'`)